src/OSPSuite.UI/Views/Charts/ChartEditorView.Designer.cs

```diff
@@ -341,116 +291,107 @@ private void InitializeComponent()
          this.layoutControlItem4.TextVisible = false;
          // 
          // layoutControlGroup2
          // 
-         this.layoutControlGroup2.CustomizationFormText = "Chart Options";
-         this.layoutControlGroup2.Items.AddRange(new DevExpress.XtraLayout.BaseLayoutItem[] {
+         this.layoutChartOptions.Items.AddRange(new DevExpress.XtraLayout.BaseLayoutItem[] {
             this.layoutControlItem2});
-         this.layoutControlGroup2.Location = new System.Drawing.Point(0, 0);
-         this.layoutControlGroup2.Name = "layoutControlGroup2";
-         this.layoutControlGroup2.Size = new System.Drawing.Size(639, 362);
-         this.layoutControlGroup2.Text = "Chart Options";
+         this.layoutChartOptions.Location = new System.Drawing.Point(0, 0);
+         this.layoutChartOptions.Name = "layoutChartOptions";
+         this.layoutChartOptions.Size = new System.Drawing.Size(534, 277);
          // 
          // layoutControlItem2
          // 
          this.layoutControlItem2.Control = this.panelChartSettings;
          this.layoutControlItem2.CustomizationFormText = "layoutControlItem2";
          this.layoutControlItem2.Location = new System.Drawing.Point(0, 0);
-         this.layoutControlItem2.MinSize = new System.Drawing.Size(6, 6);
+         this.layoutControlItem2.MinSize = new System.Drawing.Size(5, 5);
          this.layoutControlItem2.Name = "layoutControlItem2";
-         this.layoutControlItem2.Size = new System.Drawing.Size(639, 362);
+         this.layoutControlItem2.Size = new System.Drawing.Size(534, 277);
          this.layoutControlItem2.SizeConstraintsType = DevExpress.XtraLayout.SizeConstraintsType.Custom;
          this.layoutControlItem2.TextSize = new System.Drawing.Size(0, 0);
          this.layoutControlItem2.TextVisible = false;
          // 
          // layoutControlGroup4
          // 
-         this.layoutControlGroup4.CustomizationFormText = "Chart Export Options";
-         this.layoutControlGroup4.Items.AddRange(new DevExpress.XtraLayout.BaseLayoutItem[] {
+         this.layoutChartExportOptions.Items.AddRange(new DevExpress.XtraLayout.BaseLayoutItem[] {
             this.layoutControlItem5});
-         this.layoutControlGroup4.Location = new System.Drawing.Point(0, 0);
-         this.layoutControlGroup4.Name = "layoutControlGroup4";
-         this.layoutControlGroup4.Size = new System.Drawing.Size(639, 362);
-         this.layoutControlGroup4.Text = "Chart Export Options";
+         this.layoutChartExportOptions.Location = new System.Drawing.Point(0, 0);
+         this.layoutChartExportOptions.Name = "layoutChartExportOptions";
+         this.layoutChartExportOptions.Size = new System.Drawing.Size(534, 277);
          // 
          // layoutControlItem5
          // 
          this.layoutControlItem5.Control = this.panelChartExportSettings;
-         this.layoutControlItem5.CustomizationFormText = "Curve Color Grouping";
```

Comment:
The text of the Chart Export Settings was set to Curve Color Grouping... no idea why
Once again, no hard coded text in designer file